test/test_facevalues.jl

```diff
@@ -112,9 +131,11 @@ end
     # Just smoke test to make sure show doesn't error. 
     fv = FaceValues(FaceQuadratureRule{RefQuadrilateral}(2), Lagrange{RefQuadrilateral,2}())
     show(stdout, MIME"text/plain"(), fv)
+    println(stdout)
```

Comment:
Sidenote, but these shouldn't print to stdout, but to devnull, or better yet, to an IOBuffer and the output tested.